templates/_macros/split-list.jinja

```diff
@@ -0,0 +1,103 @@
+# Params
+# is_description_full_width_on_desktop: whether description has its own row on
+#   desktop vs. 50/50 split between title/description
+# is_list_full_width_on_tablet: whether list title/description each have their
+#   own row on tablet vs. 50/50 split between title/description
```

Comment:
This may be personal preference, but it could be nice to also document the available slots here